meta-balena-zeus/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,9 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+
+SRC_URI_append = " \
+	file://0001-core-Don-t-redirect-stdio-to-null-when-running-in-co.patch \
+	file://0002-remove_systemd-getty-generator.patch \
+	file://0003-Don-t-run-specific-services-in-container.patch \
+	"
```

Comment:
Same comment for the systemd patches as for plymouth above: can these be moved to meta-balena-common?

Reply:
I've pulled in plymouth and tini from upstream into meta-balena-common.
They are easy to keep around and keep the same version on all layers.

But I don't think I'll do the same for systemd. Its far more entangled with other things like kernel version etc. Keeping the same up to date systemd version running on old layers like rocko will be challenging. So systemd versions the bb will be inherited from upstream and the bbappend/patches will be in each layer.